tests/test_features.py

```diff
@@ -517,3 +517,14 @@ class Spam(betterproto.Message):
     assert spam == deepcopied
     assert spam is not deepcopied
     assert spam.baz is not deepcopied.baz
+
+
+def test_is_set():
+    @dataclass
+    class Spam(betterproto.Message):
+        foo: bool = betterproto.bool_field(1)
+        bar: Optional[int] = betterproto.int32_field(2, optional=True)
+
+    assert not Spam().is_set("foo")
+    assert not Spam().is_set("bar")
+    assert Spam(foo=True).is_set("foo")
```

Comment:
```suggestion
    assert Spam(foo=True).is_set("foo")
    assert Spam(foo=0).is_set("bar")
```

To round the cases off.